### PR TITLE
ContentMigrationBatch#S49

### PR DESCRIPTION
**Pull request by simgobb**
https://developers.google.com/web/updates/2015/05/alt-click-to-expand-all-child-nodes
https://developers.google.com/web/updates/2015/05/autocomplete-for-bracket-notation
https://developers.google.com/web/updates/2015/05/automatically-pause-on-any-exception
https://developers.google.com/web/updates/2015/05/better-incognito-devtools
https://developers.google.com/web/updates/2015/05/command-click-to-add-multiple-cursors-in-the-sources-panel

@rachelandrew

### CODE REVIEW

**Review by rachelandrew — APPROVED** [review]

File: site/en/blog/automatically-pause-on-any-exception/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+description: >
+   In the DevTools, you can automatically pause on any exception (especially useful for uncaught exceptions).
+authors:
+  - umarhansa
+date: 2015-05-16
+updated: 2015-05-19
+---
+
+<figure>
+{% Img src="image/T4FyVKpzu4WKF1kBNvXepbi08t52/mcIwVmL8QGFVDI0rvGCA.gif", alt="Automatically pause on any exception", width="614", height="390" %}
```
Comment (techeverri) [inline site/en/blog/automatically-pause-on-any-exception/index.md:13]: This links points to the GIF for the "Autocomplete for bracket notation" post

So https://developer.chrome.com/blog/automatically-pause-on-any-exception/ shows the wrong GIF

Comment (techeverri) [reply site/en/blog/automatically-pause-on-any-exception/index.md:13]: I would create a pull request but I don't know the link to the correct GIF

Comment (techeverri) [reply site/en/blog/automatically-pause-on-any-exception/index.md:13]: The correct GIF can be found in the original post in umaar.com

https://umaar.com/dev-tips/32-pause-on-exception/

Comment (techeverri) [reply site/en/blog/automatically-pause-on-any-exception/index.md:13]: I've created an issue https://github.com/GoogleChrome/developer.chrome.com/issues/2932

**Pull request merged**